FABulous/fabric_generator/file_parser.py

```diff
@@ -903,6 +900,12 @@ def parseBelFile(
                 f"NoConfigBits does not match with the BEL map in file {filename}, length of BelMap is {len(belMapDic)}, but with {noConfigBits} config bits"
             )
 
+    if individually_declared:
+        logger.warning(
+            f"Ports in {filename} have been individually declared rather than as a vector."
+        )
+        logger.warning("Ports will not be concatenated during fabric generation.")
```

Comment:
If this will actually result in an error, should we raise an exception?

Reply:
I think it is a warning is fine, since this is not an error itself, but it can lead to some errors.
This is more an issue of how the code parsing and generation is currently working.

In general, we should maybe also switch to the yosys-ghdl plugin to parse the VHDL bels.

Reply:
That will require us to update all the bels to use the VHDL attribute. It's annoying to do, but we can work on that later.

Reply:
Issue #310